src/style.css

```diff
@@ -1 +1,50 @@
 /* add styles here */
+body {
+  display: flex;
+  margin: 0;
+  background-color: gray;
+}
+
+.display {
+  display: flex;
+  margin: 0 auto;
+  padding-bottom: 6px;
+  align-items: center;
+  background-color: rgba(255, 255, 255, 1);
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.stars__star {
+  width: 12px;
+  height: 12px;
+  background-image: url(images/star.svg);
+  background-size: cover;
+  margin: 2px;
+  margin-right: 6px;
```

Comment:
Avoid setting margins on all sides of an element. It's better to choose either top or bottom for vertical spacing to prevent margin collapse issues. Since you have a right margin defined on line 28, you might want to remove the general margin on line 27, or make sure it's intentional and won't cause any layout issues.